### DISCUSSION TITLE
翻译词汇参考与讨论

### DISCUSSION THREAD

**Discussion by greyli** [apiflask/docs-zh#10] [General]
晚点会制定一些词汇的翻译对照表。如果你有不确认的地方，请发到这里。

**Comment by FarmerChillax** [comment]
schema
- 模型 👀
- 架构 ♥
- schema（不译）🚀

**Reply by FarmerChillax** [reply]
参考《高性能MySQL》一书，对 Schema 并没有进行翻译

**Comment by FarmerChillax** [comment]
Interactive API documentation

- 交互式API文档 👀
- Interactive API documentation（不译）🚀

**Comment by FarmerChillax** [comment]
validate

- 校验 👀
- 验证 ♥
- 根据上下文灵活处理 🎉
- validate（不译）🚀

**Comment by FarmerChillax** [comment]
debug mode

- Debug 模式 👀
- 调试模式 ♥
- debug mode（不译）🚀

**Comment by FarmerChillax** [comment]
origin

- 来源 👀
- origin（不译）🚀
- 看语境灵活处理  ♥

**Comment by greyli** [comment]
后续考虑怎么改进翻译「快捷路由装饰器」（route shortcuts）。

**Reply by FarmerChillax** [reply]
大致上翻译为「语法糖」如何？

- 路由装饰器语法糖
- 路由方法语法糖
- 路由语法糖

**Reply by greyli** [reply]
在第一次介绍的地方用「快捷路由装饰器」，后续直接用「路由装饰器」就行。

**Comment by FarmerChillax** [comment]
format

来源于：视图函数返回响应内容相关章节

问题：如果使用格式化，`input` 与 `output` 应该如何对应的翻译

- 格式化 🚀
- 序列化 👀

相关链接：https://segmentfault.com/a/1190000039119818
